packages/client/src/promotionEvaluations/getPromotionEvaluationItems.ts

```diff
@@ -0,0 +1,36 @@
+import client, { adaptError } from '../helpers/client';
+import join from 'proper-url-join';
+import type { GetPromotionEvaluationItems } from './types';
+
+/**
+ * Method responsible for getting the promotion evaluation items of an evaluation.
+ *
+ * @function getPromotionEvaluationItems
```

Comment:
You can remove this line since we have a const with the name of the function exported :)

Reply:
Done 🙂